frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataSidebar.jsx

```diff
@@ -372,4 +372,4 @@ function DatasetFieldMetadataSidebar({
 
 DatasetFieldMetadataSidebar.propTypes = propTypes;
 
-export default DatasetFieldMetadataSidebar;
+export default React.memo(DatasetFieldMetadataSidebar);
```

Comment:
This component rerenders many, many times, which makes Cypress flaky. Wrapping it in a `memo` handles a couple of the rerenders but not all of them. I think using `React.memo` here is fine?